src/platform/packages/shared/response-ops/rule_form/src/rule_details/rule_dashboards.tsx

```diff
@@ -121,14 +131,14 @@ export const RuleDashboards = ({ contentManagement }: Props) => {
   const loadDashboards = useCallback(async () => {
     if (contentManagement) {
       const dashboards = await dashboardServiceProvider(contentManagement)
-        .fetchDashboards()
+        .fetchDashboards({ limit: 1000, text: `${searchValue}*` })
```

Comment:
I don't see a reason to set this to 1k if we are having ES do a search on every type-ahead. 

If it is the case that ES returns a broader set of results perhaps it makes sense. WDYT?

Reply:
Optimization change in this [commit](https://github.com/elastic/kibana/pull/221848/commits/d72cf1ff534c4156229b5d3fefed4ae72cc5923c)